win32ss/user/ntuser/timer.c

```diff
@@ -219,8 +221,12 @@ IntSetTimer( PWND Window,
   {
       IntLockWindowlessTimerBitmap();
 
-      IDEvent = RtlFindClearBitsAndSet(&WindowLessTimersBitMap, 1, HintIndex);
-
+      IDEvent = RtlFindClearBitsAndSet(&WindowLessTimersBitMap, 1, HintIndex++);
+      if (IDEvent == (UINT_PTR) -1)
+	  {
```

Comment:
Please use only spaces for indentation, not tabs (all your lines below are affected too)